src/app/blog/[...slug]/page.tsx

```diff
@@ -0,0 +1,49 @@
+import { notFound } from "next/navigation";
+// eslint-disable-next-line import/no-unresolved
+import { allBlogs, Blog } from "contentlayer/generated";
```

Comment:
Do you need to generate, before running the tests?
Is there a recommendation, if the generated files should be checked in? Imo, if we don't have that many block posts it should be fine to regenerate every time, but that won't scale very well

Reply:
I have found only this https://www.contentlayer.dev/docs/getting-started-cddd76b7#ignore-build-output